docs/man/man1/kubectl-drain.1

```diff
@@ -0,0 +1,154 @@
+
+.PP
+The given node will be marked unschedulable to prevent new pods from arriving.
+Then all running pods will be removed from the machine.  If there are any
+unreplicated pods, no pods will be touched without \-\-force.
+
+.PP
+When you are ready to put the node back into service, use kubectl unlame.
+
+
+.SH OPTIONS
+.PP
+\fB\-\-force\fP=false
+    Delete unreplicated pods.
+
+.PP
+\fB\-\-grace\-period\fP=\-1
+    Period of time in seconds given to each pod to terminate gracefully. If negative, the default value specified in the pod will be used.
+
+
+.SH OPTIONS INHERITED FROM PARENT COMMANDS
+.PP
+\fB\-\-alsologtostderr\fP=false
+    log to standard error as well as files
+
+.PP
+\fB\-\-api\-version\fP=""
+    The API version to use when talking to the server
+
+.PP
+\fB\-\-certificate\-authority\fP=""
+    Path to a cert. file for the certificate authority.
+
+.PP
+\fB\-\-client\-certificate\fP=""
+    Path to a client key file for TLS.
+
+.PP
+\fB\-\-client\-key\fP=""
+    Path to a client key file for TLS.
+
+.PP
+\fB\-\-cluster\fP=""
+    The name of the kubeconfig cluster to use
+
+.PP
+\fB\-\-context\fP=""
+    The name of the kubeconfig context to use
+
+.PP
+\fB\-\-insecure\-skip\-tls\-verify\fP=false
+    If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure.
+
+.PP
+\fB\-\-kubeconfig\fP=""
+    Path to the kubeconfig file to use for CLI requests.
+
+.PP
+\fB\-\-log\-backtrace\-at\fP=:0
+    when logging hits line file:N, emit a stack trace
+
+.PP
+\fB\-\-log\-dir\fP=""
+    If non\-empty, write log files in this directory
+
+.PP
+\fB\-\-log\-flush\-frequency\fP=5s
+    Maximum number of seconds between log flushes
+
+.PP
+\fB\-\-logtostderr\fP=true
+    log to standard error instead of files
+
+.PP
+\fB\-\-match\-server\-version\fP=false
+    Require server version to match client version
+
+.PP
+\fB\-\-namespace\fP=""
+    If present, the namespace scope for this CLI request.
+
+.PP
+\fB\-\-password\fP=""
+    Password for basic authentication to the API server.
+
+.PP
+\fB\-s\fP, \fB\-\-server\fP=""
+    The address and port of the Kubernetes API server
+
+.PP
+\fB\-\-stderrthreshold\fP=2
+    logs at or above this threshold go to stderr
+
+.PP
+\fB\-\-token\fP=""
+    Bearer token for authentication to the API server.
+
+.PP
+\fB\-\-user\fP=""
+    The name of the kubeconfig user to use
+
+.PP
+\fB\-\-username\fP=""
+    Username for basic authentication to the API server.
+
+.PP
+\fB\-\-v\fP=0
+    log level for V logs
+
+.PP
+\fB\-\-vmodule\fP=
+    comma\-separated list of pattern=N settings for file\-filtered logging
+
+
+.SH EXAMPLE
+.PP
+.RS
+
+.nf
+# Drain node "foo", even if there are unreplicated pods on it.
+$ kubectl drain foo \-\-force
+
+# As above, but don't delete unreplicated pods, and use a grace period of 15 minutes.
+$ kubectl drain foo \-\-grace\-period=900
+
+
+.fi
+.RE
+
+
+.SH SEE ALSO
+.PP
+\fBkubectl(1)\fP,
```

Comment:
> add lame and unlame here?

The SEE ALSO sections are currently auto-generated based only on parent/child relationships, and it's all based on the cobra command data structure.  See https://github.com/kubernetes/kubernetes/blob/master/cmd/genman/gen_kubectl_man.go#L133

I can tweak this, but it would probably require a patch to cobra as well.  How about if I leave a TODO about this for now?